apps/students/views.py

```diff
@@ -173,3 +173,32 @@ def update_from_sheets(request):
         return redirect('students:index')
 
     return render(request, 'students/update_from_sheets.html')
+
+@login_required
+@user_passes_test(has_profile, redirect_field_name=None,
+                  login_url=reverse_lazy('accounts:complete_profile'))
+# @permissions_required
+def new_student(request):
+
+    student = Student();
```

Comment:
Why this line?

Reply:
Sir, I don't know about python classes much as I have worked in java only, so I thought of creating the instance of the class and then using it to access the villages so as to pass them into the html file.

Reply:
But you're still not using this instance anywhere. You're directly using the class name while rendering the HTML page and not the instance you created (which is the correct way).

Reply:
Ok sir made those changes.